site/app/models/gradeable/Gradeable.php

```diff
@@ -2206,4 +2206,18 @@ public function isLocked(string $user_id): bool {
         }
         return true;
     }
+    /**
+     * Returns prerequisite for a gradeable
+     *
+     * @return string
+     */
+    public function getPrerequisite(): string {
+        if ($this->depends_on !== null && $this->depends_on_points !== null) {
+            $dependent_gradeable = $this->core->getQueries()->getGradeableConfig($this->depends_on);
+            return $dependent_gradeable->getId();
```

Comment:
Should the gradeable title be displayed to students as opposed to the gradeable id?

Reply:
ya that makes more sense